File: WarGamesAPI/Model/Answer.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace WarGamesAPI.Model;

public class Answer
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Id { get; set; }
    public string? Text { get; set; }
    public DateTime Date { get; set; }

    [ForeignKey("QuestionId")]
    public int? QuestionId { get; set; }
    public Question? Question { get; set; }
} 

File: CodeRed/src/app/model/answer/answer.ts
export class Answer {
    public id: number;
    public text : string;
    public date: Date;
    public questionId: number;
    public conversationId : number;
    constructor(
        id: number,
        text : string,
        date: Date,
        questionId: number,
        conversationId : number,

    ) {
      this.id = id;
      this.text = text ;
      this.date = date;
      this.questionId = questionId;
      this.conversationId  = conversationId ;
 
    }
}
